supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiConfigurationAttributes.scala

```diff
@@ -101,16 +102,18 @@ object PipelinesApiConfigurationAttributes {
 
   def apply(googleConfig: GoogleConfiguration, backendConfig: Config): PipelinesApiConfigurationAttributes = {
 
-    def validateVPCConfig(networkOption: Option[String], authOption: Option[String]): ErrorOr[Option[VirtualPrivateCloudConfiguration]] = {
-      (networkOption, authOption) match {
-        case (Some(network), Some(auth)) =>
-          googleConfig.auth(auth) match {
-            case Valid(validAuth) => Option(VirtualPrivateCloudConfiguration(network, validAuth)).validNel
-            case Invalid(error) => s"Auth $auth is not valid for Virtual Private Cloud configuration. Reason: $error" .invalidNel
-          }
-        case (Some(_), None) => "Auth scheme not provided for Virtual Private Cloud configuration.".invalidNel
-        case (None, Some(_)) => "Network label key not provided for Virtual Private Cloud configuration.".invalidNel
-        case (None, None) => None.validNel
+    def vpcErrorMessage(missingKeys: List[String]) = s"Virtual Private Cloud configuration is invalid. Missing keys: `${missingKeys.mkString(",")}`.".invalidNel
+
+    def validateVPCConfig(networkOption: Option[String], subnetworkOption: Option[String], authOption: Option[String]): ErrorOr[Option[VirtualPrivateCloudConfiguration]] = {
+      (networkOption,subnetworkOption, authOption) match {
+        case (Some(network), _, Some(auth)) => googleConfig.auth(auth) match {
+          case Valid(validAuth) => Option(VirtualPrivateCloudConfiguration(network, subnetworkOption, validAuth)).validNel
+          case Invalid(error) => s"Auth $auth is not valid for Virtual Private Cloud configuration. Reason: $error" .invalidNel
+        }
+        case (Some(_), _, None) => vpcErrorMessage(List("auth"))
+        case (None, _, Some(_)) => vpcErrorMessage(List("network-label-key"))
+        case (None, Some(_), None) => vpcErrorMessage(List("network-label-key", "auth"))
+        case (None, None, None) => None.validNel
```

Comment:
I see you've added tests for `Some`/`Some`/`Some` and `None`/`Some`/`None`. Do we need tests for any of the other code paths (eg `Some`/`None`/`Some`?)

Reply:
The `(Some, None, Some)` already has a test from the original VPC with only network label key PR. (https://github.com/broadinstitute/cromwell/blob/412fea17b028787e4cd9051e2ef7ddc49e393805/supportedBackends/google/pipelines/common/src/test/scala/cromwell/backend/google/pipelines/common/PipelinesApiConfigurationAttributesSpec.scala#L121-L136)

Reply:
I will add test for the remaining ones 👍